sickbeard/common.py

```diff
@@ -40,6 +40,8 @@
               ]
 
 INSTANCE_ID = str(uuid.uuid1())
+if len(sickbeard.CUR_COMMIT_HASH) == 40:
+    INSTANCE_ID += '-' + sickbeard.CUR_COMMIT_HASH[:7] # Append github style short hash
```

Comment:
sickbeard config variables aren't available here yet, if i read the code correctly?